### DISCUSSION TITLE
Permissões por Conexão

### DISCUSSION THREAD

**Discussion by rodrigoskull** [rtenorioh/Press-Ticket#137] [General]
Quando adiciono múltiplas conexões e configuro o usuário para uma conexão, o mesmo visualiza mensagens de outras conexões que estão sem departamento. Acredito que o ideal é filtrar as mensagens pela conexão, até mesmo permitir o usuário a participar de múltiplas conexões, assim como é para o departamento.

**Comment by rodrigoskull** [comment]
Seria possível esta configuração?

**Comment by rtenorioh** [comment]
mas qual o status desse usuario: admin ou user?

**Reply by rodrigoskull** [reply]
Usuário com perfil de atendente. Segue screenshots. 

Se houver uma configuração de não permitir visualizar contatos de outras conexões, e/ou permitir múltiplas conexões, acredito que ficaria perfeito.

**Comment by rtenorioh** [comment]
Bom dia, estou finalizando a v1.10.0, assim que finalizar ela e liberar estarei revendo essa parte de permissões de usuários
